src/azure-cli/azure/cli/command_modules/cdn/_validators.py

```diff
@@ -17,3 +17,7 @@ def check_port_range(port, msg):
             check_port_range(origin.http_port, msg.format('HTTP', origin.name))
             check_port_range(origin.https_port, msg.format('HTTPS', origin.name))
     return True
+
+
+def validate_priority(namespace):
+    return 0 <= namespace.priority <= 1000
```

Comment:
with the validator, when priority is not in [0,1000], no error throws, right? This function only return False for the scenario that priority is not in [0,1000].

Reply:
Fixed